### PR TITLE
Update install_prereq for aarch64

### PR DESCRIPTION
**Pull request by jvonau**
Exclude armhf when installing on aarch64
https://github.com/iiab/iiab/issues/3489 has the problem report and debug information.

### CODE REVIEW

File: contrib/scripts/install_prereq [hunk 1]
```diff
@@ -192,9 +192,14 @@ check_installed_debs() {
 	for pack in "$@" ; do
 		tocheck="${tocheck} ^${pack}$ ~P^${pack}$"
 	done
+	arch=$(uname -m)
 	pkgs=$(aptitude -F '%c %p' search ${tocheck} 2>/dev/null | awk '/^p/{print $2}')
 	if [ ${#pkgs} -ne 0 ]; then
-		echo $pkgs | sed -r -e "s/ ?[^ :]+:i386//g"
+		if [ "$arch" = "X86_64" ]; then
```
Comment (holta) [inline contrib/scripts/install_prereq:198]: Uppercase `X86_64` needs to be lowercase `x86_64`

**Comment by holta** [comment]
Just FYI this PR (when revised from uppercase to lowercase `x86_64`) is tested to work on 64-bit Raspberry Pi OS — thanks to @EMG70:

- PR iiab/iiab#3523

**Comment by holta** [comment]
Please see:

- #37
- PR #38

**Comment by jcolp** [comment]
You'll need to sign the CLA, and there is already another review up for a fix for this. One of them will need to be closed. I would suggest discussing with the other individual.

**Comment by jvonau** [comment]
I'll close this one, thanks for the fix.

**Pull request closed without merging**